README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
It looks like you've added some random characters to the README. Could you please clarify the purpose of these additions?

```diff
+;wjkjewhkjeh
```